Hackney.Shared.HousingSearch/Gateways/Models/Transactions/QueryablePerson.cs

```diff
@@ -1,21 +1,20 @@
 ﻿using System;
 using Hackney.Shared.HousingSearch.Domain.Transactions;
+using Nest;
 
 namespace Hackney.Shared.HousingSearch.Gateways.Models.Transactions
 {
     public class QueryablePerson
     {
         public Person Create()
```

Comment:
I would name this something like `ToPerson(...)` because it's new really creating a `QueryablePerson`

Reply:
I agree, will fix it